### PR TITLE
Replace inline storage repos with external dependency.

### PR DESCRIPTION
**Pull request by jthomas**
Deleted storage project code now availabe in Github as NPM package. Re-factored code to use external package.
Also cleaned up un-used NPM modules using depcheck.

This PR is the first step in removing the deployer code from this project and into a separate repo (and package).

### CODE REVIEW

File: deployer/package.json [hunk 1]
```diff
@@ -54,11 +42,7 @@
     "@types/node": "11.13.4",
     "@types/randomstring": "^1.1.6",
     "@types/rimraf": "^3.0.0",
-    "@types/swagger-schema-official": "2.0.15",
```
Comment (joshuaauerbachwatson) [inline deployer/package.json:57]: You might need to restore this one, or else fix the root cause for it's being here fixed upstream.

**Review by joshuaauerbachwatson — APPROVED** [review]
Two comments.  (1) I note the checking failures ... I think the swagger dependency either needs to be reinstated or you need a fix to the nodejs client upstream.   (2) I notice that you removed the `gcsprovider` and `awsprovider` folders but not the `storage` folder.   I think all three are replaced by your new package, no?

LGTM otherwise and approving with confidence that you'll address those concerns.

File: deployer/package.json (hunk 1 above)
Comment (jthomas) [reply deployer/package.json:57]: Ah - I saw this. Having returned the dep until we fix it upstream.

**Comment by jthomas** [comment]
> (2) I notice that you removed the `gcsprovider` and `awsprovider` folders but not the `storage` folder. I think all three are replaced by your new package, no?

Thanks for catching this - now removed.

**Pull request merged**